src/main/webapp/index.html

```diff
@@ -5,6 +5,7 @@
     <title>TRACE</title>
     <link rel="stylesheet" href="index-style.css">
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB-kjcAxxbrQeJJEvSg5XgkTmGI4aZLS5E"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB-kjcAxxbrQeJJEvSg5XgkTmGI4aZLS5E&libraries=places"></script>
```

Comment:
I had issues with including both lines 7 and 8 in mine, I think just line 8 gets the job done, worth a look.